src/front-end/src/App.test.js

```diff
@@ -1,9 +1,3 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
 test('renders learn react link', () => {
```

Comment:
Minor: can you rename this to "sample test" or something like that?

Reply:
I just did!

Reply:
Thanks for mentioning that